Doc/reference/simple_stmts.rst

```diff
@@ -353,8 +353,13 @@ target, then the interpreter evaluates the target except for the last
 
 .. seealso::
 
-   :pep:`526` - Variable and attribute annotation syntax
+   :pep:`526` - Syntax for Variable Annotations
+      The proposal that added syntax for annotating variables and class
```

Comment:
The proposal that added syntax for annotating the types of variables (including class variables and instance variables), instead of expressing them through comments.